api/src/main/java/org/pmiops/workbench/monitoring/MonitoringService.java

```diff
@@ -0,0 +1,16 @@
+package org.pmiops.workbench.monitoring;
+
+import io.opencensus.stats.Aggregation;
+import io.opencensus.stats.Measure;
+
+public interface MonitoringService {
+  public void registerSignal(
+      String signalName,
+      String signalDescription,
+      Measure measurementInformation,
+      Aggregation aggregation);
+
+  public void sendLongSignal(Measure.MeasureLong measurementInformation, Long longValue);
+
+  public void sendDoubleSignal(Measure.MeasureDouble measurementInformation, Double doubleValue);
```

Comment:
If we go this route, we can define a little helper value class like `DoubleSignal` that has members for the `MeasureDouble` and `Dobule` value, and then just call a method like `sendSignal(DoubleSignal ds)` overloaded with `sendSignal(LongValue lv)`.